pkg/config/analysis/msg/messages.yaml

```diff
@@ -298,7 +298,7 @@ messages:
   - name: "ServiceEntryAddressesRequired"
     code: IST0134
     level: Warning
-    description: "Virtual IP addresses are required for ports serving TCP (or unset) protocol"
+    description: "Virtual IP addresses are required for ports serving TCP (or unset) protocol when ISTIO_META_DNS_AUTO_ALLOCATE is not set on a proxy"
```

Comment:
I think this doesn't fix the issue as users will still get warnings when the env variable is enabled.

https://github.com/istio/istio/blob/master/pkg/config/analysis/analyzers/serviceentry/protocoladdresses.go#L66 the analyzer already has a specific logic to it, but unfortunately the schema validation throws the warning.

Reply:
It doesnot matter, we cannot know whether proxy has that set. It is just a hint

Reply:
https://github.com/istio/istio/blob/master/pkg/config/analysis/analyzers/serviceentry/protocoladdresses.go#L66 only check meshconfig setting,  it is still not enough

Reply:
Makes sense, this is better than the message before